documentation/cloud-manager-guide/admin/user-management/create-user.md

```diff
@@ -0,0 +1,82 @@
+---
+keywords:
+title: Create a User
+description: Create new users
+---
+# Create a User
+
+This guide outlines the required steps to create new users and assign roles in Environment Operations Center. What a user can view and which actions they can perform are dependent on their assigned role. User roles include Tenant Administrator, Environment Administrator, and Environment User. For details on permissions for each role, see the [role based permissions](../role-based-permission/role-based-permissions.md) guide.
+
+## Getting started
+
+To create a new user, select the **Create User** button on the *Users* screen.
+
+![image description](images/create-button.png)
+
+This takes you to the *Create User* form that contains all the required input fields to create a new user. Within the form you will provide the user's details and assign their role and environments.
+
+![image description](images/create-user.png)
+
+## User details
+
+In the *User Details* section you must provide the new user's first name, last name, and email in the corresponding fields. Set the user's status by adjusting the *Status* toggle to "Active".
+
+![image description](images/create-user-details.png)
+
+All the fields in the *User Details* section are required and must be completed to submit the form. If a field is missing information, you will not be able to select the **Save** button. Hovering over the **Save** button will display a reminder that the form must be complete to create the user.
+
+![image description](images/create-save-disabled.png)
+
+Once you have completed the *User Details* section, you can proceed to the *Role Details* section to assign the user a role and one or more environments.
+
+## Role details
+
+In the *Role Details* section you can assign roles and environments to the user. Only one role can be assigned per environment, but multiple environments can be associated with one role. See the [role based permission](../role-based-permission/role-based-permissions.md) guide for role definitions and permission details.
```

Comment:
I find the way the second sentence is written to be a bit confusing. Can we make this more clear and perhaps provide an example?

For example, a user may be assigned the role of Environment User for environments A, B, and C, and then be assigned the role of Environment Administrator for environments D and E.